lib/domain/usecases/get_recovery_words_interactor.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/recovery_words/get_recovery_words_failed.dart';
+import 'package:fluffychat/domain/app_state/recovery_words/get_recovery_words_success.dart';
+import 'package:fluffychat/domain/model/recovery_words/recovery_words.dart';
+import 'package:fluffychat/domain/repository/recovery_words_repository.dart';
+import 'package:matrix/matrix.dart';
+
+class GetRecoveryWordsInteractor {
+  final RecoveryWordsRepository _recoveryWordsRepository = getIt.get<RecoveryWordsRepository>();
```

Comment:
use `final _recoveryWordsRepository = getIt.get<RecoveryWordsRepository>();` for shorter